releaseNotes/JavaScriptProvider.md

```diff
@@ -0,0 +1,33 @@
+# Azure App Configuration JavaScript Provider
+
+[Source code][source_code] | [Package (npm)][package] | [Samples][samples]
+
+## 1.0.0-preview.2 - December 15, 2023
+## Breaking Changes
+- Changed the behavior when multiple labels are filtered in one selector. In the JavaScript provider configuration settings are loaded into a `Map` containing `key` and `value`, without preserving the `label`. If multiple labels are matched in one selector, the provider cannot determine which one to use. Now it will throw an error if label filter contains `*` or `,`. [#22](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/22)
+
+## Enhancements
+- Added more information to the doc comments for better IntelliSense. [#32](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/32)
+
+## Bug Fixes
+- Fixed wrong precedence of selectors after deduplication. [#31](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/31)
+- Fixed type mismatch for `ConfigurationSettingId.label` by updating @azure/app-configuration to 1.5.0. [Azure/azure-sdk-for-js#27607](https://github.com/Azure/azure-sdk-for-js/issues/27607)
+- Fixed retry logic when host of configuration store cannot be resolved, by updating @azure/core-rest-pipeline to 1.12.2. [Azure/azure-sdk-for-js#27037](https://github.com/Azure/azure-sdk-for-js/issues/27037)

```

Comment:
I have a hard time understanding the customer impact of any of these bugs. Please check out the guidelines:

_For bug fixes, describe the symptom instead of how you fixed it, so customers can evaluate whether it's beneficial for them to take an update._

Reply:
Will add more details instead of a single sentence to describe it.

Reply:
For https://github.com/Azure/azure-sdk-for-js/issues/27607 , it's fixing a mismatched type in JS SDK, so JS provider can better/correctly work with the SDK. 
For our customers, they should sense no differences at all. So I will remove this item.